frontend/src/lib/apiClient.js

```diff
@@ -33,6 +33,12 @@ const loginUser = (data) => {
   });
 };
 
+const getCurrentUser = () => apiClient.get('api/v1/user', {
```

Comment:
I think this is the right approach, though there are some changes here on an open PR of mine that actually implements this endpoint. I think we may want to call this at the top level of the app (maybe in App.js) and thread the user through the pages/Headers.